### ISSUE TITLE
ARSample is not showing Cube

### ISSUE THREAD

**Issue by gvsharma** [artoolkit/artoolkit5#335]
I have taken code on to my local and then i run it. All the samples are running fine but the cube is not drawing and am unable to find it.
Where am i wrong? Following i s the path to the sample app that was provided by SDK.

Details:
Sample from AndroidStudioProjects->ARSample;

**Comment by ThorstenBux** [comment]
Did you follow the instructions:
https://archive.artoolkit.org/documentation/doku.php?id=4_Android:android_native

**Comment by gvsharma** [comment]
Yes. The sample projects are having '.so' files and they are loaded properly by the java code. What i found simply is a camera opened up and that it. I debugged it with breakpoints. 
There should be a cube and that'd be drawn over scene.
